test/e2e-api/main.go

```diff
@@ -21,7 +21,36 @@ import (
 )
 
 func main() {
+
+	// initialize kafka handlers
+	kafkaController := NewKafkaController()
+	http.HandleFunc("/kafka/create-topic", kafkaController.CreateTopicHandler)
+	http.HandleFunc("/kafka/delete-topic", kafkaController.DeleteTopicHandler)
+	http.HandleFunc("/kafka/list-topics", kafkaController.ListTopicsHandler)
+	http.HandleFunc("/kafka/count-topic", kafkaController.CountTopicHandler)
+	http.HandleFunc("/kafka/produce-topic", kafkaController.ProduceTopicHandler)
+	http.HandleFunc("/kafka/pump-topic", kafkaController.PumpTopicHandler)
+
+	// initialize Redis handlers
+	redisController := NewRedisController()
+	http.HandleFunc("/redis/get-msg-count-contains", redisController.GetMsgCountContains)
+	http.HandleFunc("/redis/pump-stream", redisController.PumpStream)
```

Comment:
We moved Redis Stream built-in source to udsource hence no longer using `/redis/pump-stream` in e2e. Could you remove it?